# An array of joint angles for multiple legs
LegJoints[] joints
